components/coding/profileV2/BadgesDisplayedComponent.tsx

```diff
@@ -5,16 +5,30 @@ export type BadgesDisplayedComponentProps = {
   user: User;
   earnedBadges: number;
   totalBadges: number;
+  textSize: "small" | "medium" | "large";
 };
 
 export default function BadgesDisplayedComponent({
   user,
   earnedBadges,
   totalBadges,
+  textSize,
 }: BadgesDisplayedComponentProps) {
+  let textStyle;
+  switch (textSize) {
+    case "small":
+      textStyle = "flex text-sm mg:text-sm";
```

Comment:
why is the mg: there on lines 20 and 23? Is it possibly a md?

Reply:
Great job!

Reply:
I meant md you're right!! Thanks for catching this.